application/libraries/admin/pdf.php

```diff
@@ -230,6 +230,208 @@ public function setConfig($tcpdf)
     }
 
 
+	/**
+     * Override _putXMP function to ensure version doesn't appear in metadata
+     * Put XMP data object and return ID.
+	 * @return int The object ID.
+	 * @since 5.9.121 (2011-09-28)
+	 * @protected
+	 */
+	protected function _putXMP() {
```

Comment:
Just remove https://github.com/tecnickcom/TCPDF/blob/951eabf0338ec2522bd0d5d9c79b08a3a3d36b36/tcpdf.php#L9691 ?

Reply:
It would be easier just to edit the TCPDF code but as it is vendor supplied and could be updated I've used an override